flash/tabular/__init__.py

```diff
@@ -1,8 +1,4 @@
 from flash.tabular.classification import TabularClassificationData, TabularClassifier  # noqa: F401
 from flash.tabular.data import TabularData  # noqa: F401
-from flash.tabular.forecasting.data import (  # noqa: F401
-    TabularForecastingData,
-    TabularForecastingDataFrameInput,
-    TabularForecastingInputTransform,
-)
+from flash.tabular.forecasting.data import TabularForecastingData, TabularForecastingDataFrameInput  # noqa: F401
```

Comment:
No need to import `TabularForecastingDataFrameInput` here:
```suggestion
from flash.tabular.forecasting.data import TabularForecastingData  # noqa: F401
```